include/config.h

```diff
@@ -20,6 +20,20 @@ struct CmdOptions
     std::string output_prefix;
     std::string reference_file_name;
     std::string log_file_name;
+
+    CmdOptions(std::string long_, std::string short_, std::string candidate_, std::string ref_) :
```

Comment:
Do you use this constructor somewhere within sviper?

Reply:
No this constructor is for Vaquita, for when you want to generate this struct directly without the Argument Parser.